index.html

```diff
@@ -1133,6 +1133,55 @@ <h3>Revocation Bitstring Length</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Unnecessary Correlation</h3>
+
+      <p>
+There are a number of global identifiers used in a status list entry, defined in
+Section [[[#bitstringstatuslistentry]]], that can be used across [=verifiers=]
+to correlate [=subjects=]. Some of these properties include `id`,
+`statusListIndex`, and `statusListCredential`.
+      </p>
+
+      <p>
+In some cases, when [=presenting=] a [=verifiable credential=] that contains
+other global identifiers (such as a driver's license identification number),
+using other global identifiers for the status list information does not cause
+further correlation harms since the only thing required for correlation is a
+single globally unique identifier.
+      </p>
+
+      <p>
+When these global identifiers are used in [=presentations=] that utilize
+[=selective disclosure=] or [=unlinkable disclosure=], they can violate privacy
+expectations. [=Issuers=] are urged to enable status information to be
+selectively disclosable when there is an expectation that a particular
+[=verifiable credential=] can be disclosed in a way that avoids
+correlation, such as proving that an individual is above a certain age.
+[=Verifiers=] can require that status information is revealed in situations that
+require them to know the current status of a credential, and the holder might
+then consent to revealing that information or might refuse to do so for a given
+transaction. In all cases, both [=issuers=] and [=verifiers=] are urged to
+ensure that the use of global identifiers is avoided when a particular exchange
```

Comment:
Issuers cannot know what exchanges a holder might participate in, in future, and therefore this sentence does not make sense

Reply:
It makes sense with respect to modeling considerations. Issuers should choose VC models that omit global identifiers when possible or enable selective disclosure of them. In short, issuers have to issue VCs that support selective disclosure in order for holders to be able to make use of them.

Reply:
@dlongley Whilst your comment makes sense, it does not change the sentence that does not make sense with respect to the issuer.

Reply:
suggest change
```suggestion
ensure that the use of global identifiers is avoided.
```